app/views/layouts/_admin_sidebar_index.html.haml

```diff
@@ -32,3 +32,7 @@
       = link_to(admin_revision_history_path) do
         %span.fa.fa-history
         Revision History
+    %li
+      = link_to(admin_organizations_path) do
+        %span.fa.fa-group
+        Organizations
```

Comment:
While that gem looks interesting, I don't think we need anything so "fancy", since it's just temporary. Just reading an environment variable should be enough :)